public/javascripts/moneyTracker.js

```diff
@@ -163,7 +168,12 @@ app.run(function(editableOptions) {
                 }
             };
             $scope.removePlayer = function(name){
-                $scope.playerList.splice(name,1);
```

Comment:
Im surprised we didnt catch this sooner. Simple mistake due to javascript never wanting to error.